src/Symfony/Component/Form/Test/TestCaseSetUpTearDownTrait.php

```diff
@@ -45,17 +45,11 @@ protected function tearDown(): void
      */
     trait TestCaseSetUpTearDownTrait
     {
-        /**
-         * @return void
-         */
-        private function doSetUp()
+        private function doSetUp(): void
```

Comment:
Why this change and the one on the Validator component? They don't seem related to your PR.

Reply:
I squashed the commit with all the others, but this is intended to fix the fact that PHPUnit 8 needs the type-hint, and not PHPUnit 7.

As Symfony needs PHP 7.1, the void return type can be added with no harm, and since these traits are new as of the upcoming v4.3, IMO they can benefit from this return type right from the start.

I may separate the fix into another commit or into another PR if needed, but since I also need to add such trait in the new `WebTestAssertions`, I thought it would be also necessary to bring the fix to other traits at the same place. 

WYDT? Keep it as-is? New commit/PR?

Reply:
For 4.2 and should be applied to all similar traits if there are more similar cases.

Reply:
So I should split the change and put it in another PR targeted on 4.2?

Reply:
So, let's remove this change from here and let's create another PR for 4.2.